contrib/inventory/azure_rm.py

```diff
@@ -397,6 +414,14 @@ def _get_credentials(self, params):
             self.log('Retrieved default profile credentials from ~/.azure/credentials.')
             return default_credentials
 
+        try:
+            cli_credentials = self._get_azure_cli_credentials()
+            if cli_credentials:
+                self.log('Retrieving credentials from AzureCLI profile')
```

Comment:
I'd put this log earlier, but that's me

Reply:
Here is the auto detection part. I can pass the logic if CLI not available.